scripts/compile_parsers.makefile

```diff
@@ -22,25 +18,21 @@ else
    rmf         = rm -rf $(1)
 endif
 
-ifneq ($(wildcard $(SRC_DIR)/*.cc),)
+ifneq ($(wildcard src/*.cc),)
```

Comment:
is this change needed ?

Reply:
actually all the removed substitutions ?

Reply:
Acutally, src was hard-coded in the targets anyways. `$(SRC_DIR)` might just be a risk that someone actually has this defined. but I changed it now to use `$(SRC_DIR)` which might be a bad idea